config/initializers/route_config.rb

```diff
@@ -30,7 +30,7 @@
       config.host_locales[hostname] = locale
     end
   else
-    host = ENV['LOCALHOST'] || 'localhost'
+    host = ENV.fetch('LOCALHOST')
```

Comment:
Note that having default values in `.env` allows us to be more confident in our usage of `ENV.fetch` which allows us to fail fast with a readable error message when an ENV var isn't available rather than going on a `nil` hunt (https://thoughtbot.com/blog/testing-and-environment-variables#explicitly-access-keys-with-envfetch).

Reply:
👍